### PR TITLE
[Merged by Bors] - chore: fix spelling mistakes

### PR DESCRIPTION
**Pull request by euprunin**

### CODE REVIEW

File: Mathlib/Analysis/Normed/Ring/IsPowMulFaithful.lean [hunk 1]
```diff
@@ -58,7 +58,7 @@ theorem contraction_of_isPowMul {α β : Type*} [SeminormedRing α] [SeminormedR
   contraction_of_isPowMul_of_boundedWrt (SeminormedRing.toRingSeminorm α) hβ hf x
 
 /-- Given two power-multiplicative ring seminorms `f, g` on `α`, if `f` is bounded by a positive
-  multiple of `g` and viceversa, then `f = g`. -/
+  multiple of `g` and vice-versa, then `f = g`. -/
```
Comment (vihdzp) [inline Mathlib/Analysis/Normed/Ring/IsPowMulFaithful.lean:61]: I think the proper spelling is "vice versa" with no hyphen

**Review by grunweg — COMMENTED** [review]
Thanks!
maintainer merge

**Comment by grunweg** [comment]
There's a merge conflict now; can you merge `master`, please? Thanks!
awaiting-author

File: Mathlib/Tactic/Module.lean [hunk 2]
```diff
@@ -483,7 +483,8 @@ partial def reduceCoefficientwise {R : Q(Type u)} {_ : Q(AddCommMonoid $M)} {_ :
     let pf : Q(NF.eval $(l₁.toNF) = NF.eval $(l₁.toNF)) := q(rfl)
     pure ([], pf)
   /- if one of the lists is empty and the other one is not, recurse down the nonempty one,
-    forming goals that each of the listed coefficents is equal to zero -/
+    forming goals that each of the listed coefficients is equal to
+    zero -/
```
Comment (urkud) [inline Mathlib/Tactic/Module.lean:487]: Why add a newline here?

Comment (urkud) [reply Mathlib/Tactic/Module.lean:487]: Otherwise LGTM. Thanks!
bors d+

**Pull request closed without merging**